ui/raidboss/data/03-hw/alliance/weeping_city.js

```diff
@@ -0,0 +1,329 @@
+      suppressSeconds: 10,
+      response: Responses.tankCleave(),
+    },
+    {
+      id: 'Weeping City Aura Burst',
+      regex: /Aura Burst/,
+      beforeSeconds: 4,
+      condition: Conditions.caresAboutAOE(),
+      response: Responses.aoe(),
+    },
+  ],
+  triggers: [
+    {
+      // 2 of the 4 encounters in Weeping City use the 0017 head marker.
+      // 2 of the 4 use the 003E head marker.
+      // Because of this, we restrict those triggers for each boss to activate
+      // only when that boss is in progress.
+      id: 'Weeping City HeadMarker Arachne',
+      netRegex: NetRegexes.message({ line: '.*Queen\'s Room will be sealed off.*?', capture: false }),
+      run: function(data) {
+        data.arachneStarted = true;
+      },
+    },
+    {
+      id: 'Weeping City HeadMarker Ozma',
+      netRegex: NetRegexes.message({ line: '.*Gloriole will be sealed off.*?', capture: false }),
+      run: function(data) {
+        data.arachneStarted = false;
+        data.ozmaStarted = true;
+      },
+    },
+    {
+      id: 'Weeping City HeadMarker Calofisteri',
+      netRegex: NetRegexes.message({ line: '.*Tomb Of The Nullstone will be sealed off.*?', capture: false }),
+      run: function(data) {
+        data.ozmaStarted = false;
+        data.calStarted = true;
+      },
+    },
+    {
+      id: 'Weeping City Sticky Wicket',
+      netRegex: NetRegexes.headMarker({ id: '003C', capture: false }),
+      suppressSeconds: 10,
+      response: Responses.spread(),
+    },
+    {
+      id: 'Weeping City Shadow Burst',
+      netRegex: NetRegexes.headMarker({ id: '003E' }),
+      condition: function(data) {
+        return data.arachneStarted;
+      },
+      response: Responses.stackOn(),
+    },
+    {
+      id: 'Weeping City Frond Affeared',
+      netRegex: NetRegexes.startsUsing({ id: '183A', source: 'Arachne Eve', capture: false }),
+      response: Responses.lookAway(),
+    },
+    {
+      id: 'Weeping City Arachne Web',
+      netRegex: NetRegexes.headMarker({ id: '0017' }),
+      condition: function(data, matches) {
+        return data.arachneStarted && data.me == matches.target;
+      },
+      infoText: {
+        en: 'Drop Web Outside',
+      },
+    },
+    {
+      id: 'Weeping City Brand Of The Fallen',
+      netRegex: NetRegexes.headMarker({ id: '0037' }),
+      condition: Conditions.targetIsYou(),
+      response: Responses.doritoStack(),
+    },
+    {
+      id: 'Weeping City Dark Eruption',
+      netRegex: NetRegexes.headMarker({ id: '0019' }),
+      condition: Conditions.targetIsYou(),
+      infoText: {
+        en: 'Puddles on YOU',
+        de: 'Pfützen auf DIR',
+        fr: 'Mare sur VOUS',
+        ja: '自分に床範囲',
+        cn: '点名',
+        ko: '장판 바깥에 깔기',
+      },
+    },
+    {
+      id: 'Weeping City Beguiling Mist',
+      netRegex: NetRegexes.startsUsing({ id: '17CE', source: 'Summoned Succubus' }),
+      condition: function(data) {
+        return data.canSilence();
```

Comment:
canSilence => CanSilence

Reply:
That's incredibly annoying, I specifically remember fixing that.